rpi-eeprom-config

```diff
@@ -97,10 +97,23 @@ class BootloaderImage(object):
                 sys.stdout.write(config_bytes)
 
 def main():
-    parser = argparse.ArgumentParser('RPI EEPROM config tool')
+    parser = argparse.ArgumentParser(formatter_class=argparse.RawDescriptionHelpFormatter, \
+    description='RPI EEPROM config tool. \
+\n\nThere are 3 operating modes: \
+\n\n1. Output the bootloader configuration stored in an EEPROM image file to \
+the screen (STDOUT): specify only the name of an EEPROM image file using the \
+EEPROM option. \
+\n\n2. Output the bootloader configuration stored in an EEPROM image file to a \
+file: specify the EEPROM image file using the EEPROM option, and the output file\
+using the --out option. \
+\n\n3. Insert a new bootloader configuration into an EEPROM image file: specify \
+the EEPROM image file using the EEPROM option, the bootloader configuration file\
+using the --config option, and the name to use for the new file which will be a \
+combination of the original EEPROM image file, along with the new bootloader \
+configuration.')
     parser.add_argument('--config', help='Filename of new bootloader config')
-    parser.add_argument('--out', help='Filename for the EEPROM image with updated config')
-    parser.add_argument('eeprom', help='EEPROM filename (pieeprom.bin)')
+    parser.add_argument('--out', help='Name of output file')
+    parser.add_argument('EEPROM', help='EEPROM filename (pieeprom.bin)')
```

Comment:
I think this should be '--eeprom'. Currently the unit test fails

cd  test
./test-rpi-eeprom-config

check_update ../firmware/critical/pieeprom-2019-07-15.bin bootconf-2019-07-15.txt
Traceback (most recent call last):
  File "/home/tim/rpi-eeprom/test/../rpi-eeprom-config", line 126, in <module>
    main()
  File "/home/tim/rpi-eeprom/test/../rpi-eeprom-config", line 119, in main
    image = BootloaderImage(args.eeprom, args.out)
AttributeError: 'Namespace' object has no attribute 'eeprom'
ERROR
Config-read: checksum mismatch

Reply:
Sorry that was me - I was toying with the idea of putting the EEPROM parameter in upper case in order to make the help description slightly more readable, but I decided against it, then forgot I had already changed that. It's back the way it was now - 'eeprom' in lower case, which is correct. (There is no --eeprom parameter).